src/transform.js

```diff
@@ -6,6 +6,14 @@ const computeNewlines = require("./compute-newlines.js");
 const locToString = loc =>
   `${loc.start.line}:${loc.start.column}-${loc.end.line}:${loc.end.column}`;
 
+const stripSuffixFromImportSource = path => {
+  // TODO: make this configurable so we can output .ts[x]?
```

Comment:
Can you do `import foo from "./foo.ts"`?  In the past when I've tried it complained so ended up always using `import foo from "./foo"` when writing TypeScript.

Reply:
I've successfully used `export foo from "./foo.ts"`, but that said, in this case I changed the flow files to "export *" to get rid of errors.

To double-check, are you suggesting that the export should be rewritten as following?

```
import { foo } from "./foo";
export { foo };
```

Or something else?

Reply:
I wasn't suggesting any changes to this PR.  Just curious about the TODO.

Reply:
Can create an issue for the TODO?

Reply:
I didn't realize that I wrote this TODO and you were just moving the code.  I can create a ticket for it.  😅